support-frontend/assets/components/checkoutNudge/checkoutNudgeContainer.tsx

```diff
@@ -46,15 +35,13 @@ export function CheckoutNudgeContainer({
 			  (weeklyMinAmount / 100)
 					.toFixed(weeklyMinAmount % 100 === 0 ? 0 : 2)
 					.toString();
-	const subTitleStart =
-		nudgeMinVariantA || nudgeMinVariantB ? `for` : `from just`;
-	const [title, subtitle, paragraph] = [
-		`Support us every year`,
-		`${subTitleStart} ${
-			currencyGlyph + minAmount.toString()
-		} (${minWeeklyAmount} a week)`,
-		`Funding Guardian journalism every year doesn’t need to be expensive. Make a bigger impact today, and protect our independence long term. Please consider annual support.`,
-	];
+
+	const title = 'Support us every year';
+	const subtitle = `for ${
```

Comment:
Think subtitle should read 'with £50 if you can (£1 per week)', will tweak it now